src/wikibots/pas.py

```
@@ -0,0 +1,114 @@
+from typing import Any
+
+import requests
+from mwparserfromhell.nodes import ExternalLink
+from pywikibot import warning, info
+from pywikibot.pagegenerators import PagesFromTitlesGenerator, SearchPageGenerator
+
+try:
+    sys.path.append(os.path.dirname(os.path.abspath(__file__)) + '/lib')
+    from lib.bot import BaseBot
+    from lib.wikidata import WikidataProperty
+except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)